trl/trainer/utils.py

```diff
@@ -898,8 +900,9 @@ def get_kbit_device_map() -> Optional[dict[str, int]]:
         return None
 
 
-def get_peft_config(model_config: ModelConfig) -> "Optional[PeftConfig]":
-    if model_config.use_peft is False:
+@deprecate_kwarg("model_config", "0.14.0", "model_args", warn_if_greater_or_equal_version=True)
```

Comment:
I'm pretty sure no one uses this function with this keyword argument syntax; (use argument form instead)